Source/ApparentHorizonFinder/AHInitialGuess.hpp

```diff
@@ -128,29 +128,32 @@ class AHInitialGuessMerger : public AHInitialGuessDefault
 class AHInitialGuessEllipsoid : public AHInitialGuessDefault
 {
   public:
-    double m_radius_x, m_radius_y;
-#if CH_SPACEDIM == 3
-    double m_radius_z;
-#endif
+    double D_DECL(m_radius_x, m_radius_y, m_radius_z);
 
     AHInitialGuessEllipsoid() {}
 
 #if CH_SPACEDIM == 3
     AHInitialGuessEllipsoid(double a_radius_x, double a_radius_y,
                             double a_radius_z)
+    AHInitialGuessEllipsoid(D_DECL(double a_radius_x, double a_radius_y,
+                            double a_radius_z))
```

Comment:
This seems to have been messed up a bit. The first 3 lines should be removed.
```suggestion
    AHInitialGuessEllipsoid(D_DECL(double a_radius_x, double a_radius_y,
                            double a_radius_z))
```